api/router.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"github.com/numary/ledger/api/controllers"
+	"github.com/numary/ledger/api/middlewares"
+	"github.com/numary/ledger/ledger"
+)
+
+// Router -
+func Router(cc cors.Config, resolver *ledger.Resolver) *gin.Engine {
+	router := gin.Default()
+
+	// Default Middlewares
+	router.Use(
+		cors.New(cc),
+		gin.Recovery(),
+		middlewares.AuthMiddleware(router),
+	)
+
+	// API Controllers
+	configController := controllers.CreateConfigController()
+	ledgerController := controllers.CreateLedgerController()
+	transactionController := controllers.CreateTransactionController()
+	accountController := controllers.CreateAccountController()
+	scriptController := controllers.CreateScriptController()
+
+	// API Routes
+	router.GET("/_infos", configController.GetInfos)
```

Comment:
Actual route is `_info`, it's possible to declare the same route ?

Reply:
Yes sure! The change is now on that pull request.